packages/moonstone/VirtualList/VirtualList.js

```diff
@@ -40,16 +42,25 @@ const VirtualList = kind({
 		 * <VirtualList itemSize={ri.scale(72)} />
 		 * ```
 		 *
-		 * @type {Number}
+		 * @type {Number|ui/VirtualList.itemSizesShape}
 		 * @required
 		 * @public
 		 */
-		itemSize: PropTypes.number.isRequired
+		itemSize: PropTypes.oneOfType([PropTypes.number, itemSizesShape]).isRequired
```

Comment:
This is an API change which makes this a difficult PR to accept right now. We may need to create a new private component export if we want to experiment with this now.